pages/availability/event/[type].tsx

```diff
@@ -13,6 +13,7 @@ import {
   XIcon,
   PhoneIcon,
 } from '@heroicons/react/outline';
+import dayjs from 'dayjs';
```

Comment:
Codacy has a fix for the issue: [Replace `'dayjs'` with `"dayjs"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
import dayjs from "dayjs";
```